temporian/core/operators/window/base.py

```diff
@@ -68,6 +68,7 @@ def __init__(
                     "`window_length` must have exactly one float64 feature."
                 )
             self.add_input("window_length", window_length)
+            self._window_length = None
```

Comment:
good catch :) and great example of the problems associated with having tons of tests on the implementation and not so tons of them on the core op